.ci/update-github-io.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-github-io.sh <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+
+TAG=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+echo TAG="$TAG"
+git checkout "$TAG"
```

Comment:
????

Version is defined by user. `git checkout checkstyle $TARGET_VETSION`

Reply:
Changes Done.